crypto/marshal.go

```diff
@@ -29,18 +31,22 @@ func ParseKeyFile(keyFilePath string) (elliptic.Curve, []byte, error) {
 		}
 
 		if block.Type == "EC PRIVATE KEY" {
-			break
+			privKey, err := x509.ParseECPrivateKey(block.Bytes)
```

Comment:
So this will parse PEM encoded keys all from the same file?
Should we also support using multiple files?

Reply:
This could potentially be problematic from a deployment standpoint, but I think we can do without multiple file support.